CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Remember to align the itemized text with the first line of an item within a list
 * New Functionality
   * Added {func}`jax.numpy.unstack`, following the addition of this function in
     the array API 2023 standard, soon to be adopted by NumPy.
+  * Added {func}`jax.numpy.cumulative_sum`, following the addition of this
```

Comment:
You could put both of these under one bullet point to keep the changelog more concise